### DISCUSSION TITLE
How can I fix this error: (ERROR  Cannot start nuxt:  Cannot find module 'nuxt3') when trying to run a nuxt app in a monorepo

### DISCUSSION THREAD

**Discussion by glopezep** [nuxt/nuxt#17196] [Questions]
I think this is happening because the nuxt app is not in the root, this was working in the RC 3, so i just need to know if I have to add some configuration

**Comment by danielroe** [comment]
Likely you are using an outdated version of nuxt. Since the RC release, you should import from `'nuxt'` and not `'nuxt3'`.

**Reply by glopezep** [reply]
@danielroe I'm using this version:  "nuxt": "3.0.0-rc.6"

**Reply by danielroe** [reply]
Would you provide a reproduction?

**Reply by glopezep** [reply]
sure
